File: tutorials/csharp/remote_file/rf_client/RemFile.cs

using System;
using SocketProAdapter;
using SocketProAdapter.ClientSide;
using System.IO;

public class RemotingFile : CAsyncServiceHandler
{
    public RemotingFile()
        : base(RemFileConst.sidRemotingFile)
    {
        m_sh = new CStreamHelper(this);
    }

    public CStreamHelper StreamHelper
    {
        get
        {
            return m_sh;
        }
    }
    private CStreamHelper m_sh;
}

File: tutorials/vbnet/remote_file/rf_client/RemFile.vb
Imports System
Imports SocketProAdapter
Imports SocketProAdapter.ClientSide
Imports System.IO

Public Class RemotingFile
    Inherits CAsyncServiceHandler

    Public Sub New()
        MyBase.New(RemFileConst.sidRemotingFile)
        m_sh = New CStreamHelper(Me)
    End Sub

    Public ReadOnly Property StreamHelper() As CStreamHelper
        Get
            Return m_sh
        End Get
    End Property
    Private m_sh As CStreamHelper
End Class
